operator/backrest/restore.go

```diff
@@ -313,7 +314,10 @@ func CreateRestoredDeployment(restclient *rest.RESTClient, cluster *crv1.Pgclust
 		if err != nil {
 			log.Error("unable to marshall affinity obtained from restore job spec")
 		}
-		affinityStr = "\"affinity\":" + string(affinityBytes) + ","
+		// Since the template for a cluster deployment contains the parentheses for the json
+		// defining any affinity rules, we trim them here from the affinity json obtained directly
+		// from the restore job (which also has the same parentheses)
```

Comment:
I'd call these curly brackets or braces. Looks like JSON calls them braces.

Reply:
Good catch, changed to `braces`